tango_simlib/tests/test_tango_sim_generator.py

```diff
@@ -95,6 +96,18 @@ def test_device_command_list(self):
                           "The commands specified in the xmi file are not present in"
                           " the device")
 
+    def test_device_property_list(self):
+        """Testing whether the device properties in the model are added to
+        the TANGO sim device as expected
+        """
+        db = PyTango.Database()
```

Comment:
@Athanaseus and I had a look at this.  It won't be possible to mock, as the device server is running in a different process to the tests.  Suggest this test is removed.

We also realised that the device server shouldn't try and access the database if launched in the "nodb" mode, like we do for the tests.